log-example/src/main/resources/log4j.properties

```diff
@@ -12,7 +12,7 @@ log4j.appender.file.layout=org.apache.log4j.PatternLayout
 log4j.appender.file.layout.conversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{2}: %m%n
 
 log4j.appender.http=io.phdata.pulse.log.HttpAppender
-log4j.appender.http.Address=http://localhost:8200/v2/events/pulse-test-default
+log4j.appender.http.Address=http://edge2.valhalla.phdata.io:9015/v2/events/pulse-test-default
```

Comment:
It's fine to put this here for testing just don't check it in

Reply:
reverted